packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -37,18 +40,29 @@ const renderItems = (items, getCurrentPageProps) => {
   });
 };
 
+export interface IBreadCrumbProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+}
+
 /** @component */
-export default function Breadcrumb({ children, ...breadcrumbProps }) {
+const Breadcrumb: React.FunctionComponent<IBreadCrumbProps> = ({
```

Comment:
Since this element is publicly consumable, this should be using `React.forwardRef`